ibm_catalog.json

```diff
@@ -0,0 +1,501 @@
+							]
+						}
+					],
+					"architecture": {
+						"diagrams": [
+							{
+								"diagram": {
+									"caption": "All inclusive Cloudability Onboarding",
+									"type": "image/svg+xml",
+									"url_proxy": {
+										"url": "https://github.com/terraform-ibm-modules/terraform-ibm-apptio-cloudability-onboarding/blob/main/reference-architecture/cloudability-all-inclusive-onboarding.svg"
+									}
+								},
+								"description": "Billing exports are written to a Key Protect encrypted COS bucket. Apptio Cloudability is granted access to read from this bucket to add ibm cloud billing reports daily. Key Protect and Cloud Object Storage are created in your desired region and resource group. Optionally, configure retention and archiving of the data in your bucket and enable monitoring and auditing to your cloud object storage bucket."
+							}
+						],
+                        "features": [
+							{
+                                "title": "Daily exports of your cloud billing reports to a COS Bucket",
+                                "description": "Cloudability reads CSV Billing reports which are added to a COS bucket daily for each account in your enterprise."
+                            },
+                            {
+                                "title": "Encrypted Bucket with Key Protect",
+                                "description": "Daily billing reports are stored in a Key Protect encrypted bucket allowing the automated rotatation of encryption keys. See https://cloud.ibm.com/docs/cloud-object-storage?topic=cloud-object-storage-tutorial-kp-encrypt-bucket."
+                            },
+							{
+                                "title": "Least Privileged Operations to Cloudability",
+                                "description": "Custom access roles are created so that cloudability only has the minimal required access to your IBM Cloud Account."
+                            },
+							{
+                                "title": "Cloudability Onboarding Verification",
+                                "description": "Verification of the account onboarding to Apptio Cloudability is performed so that you have confidence that the configuration is correct."
+                            },
+							{
+                                "title": "Add all accounts in an Enterprise to Cloudability",
+                                "description": "Running this deployable architecture at the root (primary) Enterprise Account will cause all existing and future accounts in the enterprise to be added to Apptio Cloudability."
+                            },
+							{
+                                "title": "Managed with Code!",
+                                "description": "Easily re-configure or clean-up the onboarding of your accounts to Cloudability and ensure that everything is centrally connected and configured."
+                            }
+						]
+					},
+					"configuration": [
```

Comment:
By duplicating all the variable descriptions and default values in here, your going to have to maintain them in both the terraform code and this file. If you remove the descriptions and default values from here, they will get pulled from the .tf code, so I suggest you do that. 

Infact the only reason you actually should add inputs to this file is if you want to customise them (for example with dropdown lists like you have), and for the ordering. The ordering here is what you will see in the Projects UI, so I suggest to group related inputs together.

Reply:
Thanks, and yes I do want to control which inputs get exposed in the DA vs the DA code and to make use of more advanced functionality such as option inputs so I will keep this section. I will consider the ordering though to prioritize the most important inputs and logically grouping the inputs.